packages/cxl-ui/src/components/cxl-playbook-accordion.js

```diff
@@ -0,0 +1,156 @@
+import { customElement } from 'lit-element';
+import '@conversionxl/cxl-lumo-styles';
+import '@vaadin/vaadin-accordion';
+import '@vaadin/vaadin-checkbox';
+import { registerStyles } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
+import cxlPlaybookAccordionGlobalStyles from '../styles/global/cxl-playbook-accordion-css.js';
+import { CXLVaadinAccordion } from './cxl-vaadin-accordion';
+import vaadinAccordionPanelStyles from '../styles/cxl-playbook-accordion/vaadin-accordion-panel-css.js';
+import vaadinCheckboxStyles from '../styles/cxl-playbook-accordion/vaadin-checkbox-css.js';
+
+/**
+ * Allows opening multiple panels simultaneously.
+ * Saves panel state, restores on page load.
+ */
+@customElement('cxl-playbook-accordion')
+export class CXLPlaybookAccordion extends CXLVaadinAccordion {
+  /**
+   *
+   * @return {NodeListOf<Element>}
+   */
+  get accordionPanels() {
+    return this.querySelectorAll('vaadin-accordion-panel');
+  }
+
+  /**
+   *
+   * @return {NodeListOf<Element>}
+   */
+  get panelSummarySlotCheckboxes() {
+    return Array.from(this.accordionPanels).map((el) =>
+      el.querySelector('[slot="summary"] vaadin-checkbox')
+    );
+  }
+
+  /**
+   *
```

Comment:
All doccomments seem to have an empty star line for some reason?

Reply:
@lkraav default IDE behavior when I add the multiline comment:
https://www.loom.com/share/ee0045c42d5748dabcc0ea8713f7d867
If you want to force the particular multiline comment formatting why is this not auto fixed like the other formatting issues via `ESLint rules`?
I saw a specific one for this: https://eslint.org/docs/rules/multiline-comment-style
Otherwise, all the developers should use the same IDE or change the settings of their default IDE behavior.